staging/prometheus-operator/crds/crd-prometheuses.yaml

```diff
@@ -0,0 +1,4420 @@
+# https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.44.0/example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml
```

Comment:
there is a pretty sizable diff between this and what's upstream (looking at diff https://gist.github.com/gracedo/ec31fc6873a35c5f08e115955afb61f8), just wondering why? is the upstream crd mis-formatted?

Reply:
https://github.com/prometheus-community/helm-charts/blob/kube-prometheus-stack-12.11.3/charts/kube-prometheus-stack/crds/crd-prometheuses.yaml

it is, and so the upgrade path suggested is no longer valid.